paywall/src/_server.js

```diff
@@ -19,9 +19,9 @@ function _server(port, dev) {
 
           // assigning `query` into the params means that we still
           // get the query string passed to our application
-          if (pathname.match('/0x')) {
+          if (pathname.match('/0x') || pathname === '/') {
```

Comment:
If I were to split the PR, this is literally the only change that would be in the 2nd PR.

Reply:
I know... but it is still cleaner? also very easy to actually do with git?